### ISSUE TITLE
[a11y] Cannot edit record sets

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1179]
**Describe the bug**
Record sets cannot be renamed or have remarks added because the dialog does not appear.

**To Reproduce**
1. Click on Record Sets
2. Click on the pencil icon
3. See no window

**Screenshots**
<img width="528" alt="image" src="https://user-images.githubusercontent.com/37256050/158602799-92676d36-8ca2-402a-9516-07b2fb877d5f.png">

**Desktop (please complete the following information):**
 - Windows 10 & macOS 12.3, Safari, Edge on Windows & Mac

**Comment by maxxxxxdlp** [comment]
Fixed
